src/components/ConfirmationBox/ConfirmationBox.js

```diff
@@ -7,10 +7,10 @@ const ConfirmationBox = props => {
 			<div>{`Did you take ${props.item.name}?`}</div>
 			{props.children}
 			<div>
-				<button id="YES" onClick={props.onYes}>
+				<button testID="YES" onClick={props.onYes}>
```

Comment:
Unfortunately testID isn't a valid attribute for button elements, we can prefix things with data- to define our own custom attributes.

The usual convention for this is `data-test-id` (kebab case)